pod/perlfunc.pod

```diff
@@ -931,24 +931,40 @@ X<bless>
 
 =for Pod::Functions create an object
 
-This function tells the thingy referenced by REF that it is now an object
-in the CLASSNAME package.  If CLASSNAME is an empty string, it is
-interpreted as referring to the C<main> package.
-If CLASSNAME is omitted, the current package
-is used.  Because a L<C<bless>|/bless REF,CLASSNAME> is often the last
-thing in a constructor, it returns the reference for convenience.
-Always use the two-argument version if a derived class might inherit the
-method doing the blessing.  See L<perlobj> for more about the blessing
-(and blessings) of objects.
-
-Consider always blessing objects in CLASSNAMEs that are mixed case.
-Namespaces with all lowercase names are considered reserved for
-Perl pragmas.  Builtin types have all uppercase names.  To prevent
-confusion, you may wish to avoid such package names as well.
-It is advised to avoid the class name C<0>, because much code erroneously
-uses the result of L<C<ref>|/ref EXPR> as a truth value.
-
-See L<perlmod/"Perl Modules">.
+C<bless> tells Perl to mark C<REF> as an object in a package.
+The two-argument version of C<bless> is always preferred unless there is a specific reason to I<not> use it.
+
+=over
+
+=item * Bless a reference into a specific package (recommended form):
+
+    bless $ref, $package;
+
+The two-argument form adds the object to the package specified as the second argument.
+
+=item * Bless a reference into package C<main>:
+
+    bless $ref, "";
+
+If the second argument is an empty string, C<bless> adds the object to package C<main>.
+
+=item * Bless a reference into the current package (not inheritable):
+
+    bless $ref;
+
+If C<bless> is used without its second argument, the object is created in the current package. The second argument should always be supplied if a derived class might inherit a method executing C<bless>. Because it is a potential source of bugs, one-argument C<bless> is discouraged.
+
+=back
+
+See L<perlobj> for more about the blessing (and blessings) of objects.
+
+L<C<bless>|/bless REF,CLASSNAME> returns its first argument, the supplied reference, as the value of the function; since C<bless> is commonly the last thing executed in constructors, this returns the reference to the object by default and allows the caller to immediately use this returned object in method calls.
+
+C<CLASSNAME> should always be a mixed-case name, as all-uppercase and all-lowercase names are meant to be used only for Perl builtin types and pragmas, respectively. Avoid creating all-uppercase or all-lowercase package names to prevent confusion.
+
+Also avoid <Cbless>ing things into the class name C<0>; this will cause code which (erroneously) checks the result of L<C<ref>|/ref EXPR> to see if a reference is C<bless>ed to fail, as "0", a falsy value, is returned.
+
+See L<perlmod/"Perl Modules"> for more details.
```

Comment:
this is going to fail podcheck as the lines need to be under 78 characters when formatted (they will end up needing to be shorter since this is in the middle of a list in perlfunc).

Reply:
You marked this resolved, but there's still very long lines.

Reply:
I ran it through podcheck without issues, but I'll shorten everything to 64 characters.

Reply:
I see the issue; there are lines other than this specific one that are longer that that; checking every line.

Reply:
All lines trimmed to 64 characters, plus or minus a few. The really long lines are all definitely gone.